ui/screens/levelselect.lua

```diff
@@ -195,10 +208,8 @@ local function make_pack_elements()
                         )
                         cache_folder_flex[pack.id] = update_element(levels, root, 2, last_levels)
                     end
-                    if root.elements[2] ~= levels then
-                        levels.elements[1]:click(false)
-                    end
                     root.elements[2] = levels
+					levels.elements[1]:click(false)
```

Comment:
You need to keep the check, if you always autoclick on the first level when selecting a pack clicking on the pack again will start the first level. The check there is making it not autoclick if the level column is still the same.